botocore/args.py

```diff
@@ -543,6 +550,52 @@ def _compute_connect_timeout(self, config_kwargs):
         if connect_timeout:
             config_kwargs['connect_timeout'] = connect_timeout
 
+    def _compute_request_compression_config(self, config_kwargs):
+        min_size = config_kwargs.get('request_min_compression_size_bytes')
+        disabled = config_kwargs.get('disable_request_compression')
+        if min_size is None:
+            min_size = self._config_store.get_config_variable(
+                'request_min_compression_size_bytes'
+            )
+            config_kwargs['request_min_compression_size_bytes'] = min_size
+        # conversion func is skipped so input validation must be done here
+        # regardless if the value is coming from the config store or the
+        # config object
+        self._validate_min_compression_size(min_size)
+        if disabled is None:
+            disabled = self._config_store.get_config_variable(
+                'disable_request_compression'
+            )
+        else:
+            # if the user provided a value we must check if it's a boolean
+            disabled = ensure_boolean(disabled)
+        config_kwargs['disable_request_compression'] = disabled
+
+    def _validate_min_compression_size(self, min_size):
+        min_allowed_min_size = 0
```

Comment:
What was the final decision on this? Are we actually creating larger bodies for empty bodies?

Reply:
I got push back when I asked about this offline. I'm double checking with the owner of the spec for confirmation.  However, it does look like the Ruby SDK, which released compression a few weeks ago, has the absolute minimum at 0:

https://github.com/aws/aws-sdk-ruby/blob/ee2f883b8766aded0b86b3ee2b27575c90d9f3ac/gems/aws-sdk-core/lib/aws-sdk-core/plugins/request_compression.rb#L50

Reply:
Seems as though we're updating the spec for the minimum to be at least 1. I'll update this accordingly